build/android/config.gni

```diff
@@ -71,6 +71,8 @@ brave_chrome_java_deps += brave_app_playlist_deps
 brave_chrome_all_java_deps = [
   "//brave/android/features/tab_ui:java",
   "//brave/browser/partnercustomizations:delegate_public_impl_java",
+
+  #"//components/search_engines/android:delegate_public_impl_java",
```

Comment:
cleanup this?

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/25340/commits/3c162e5149898e3f8d960842fb080488a8b8b5dd